examples/tabbed-document-ui/src/widgets/tab_bar.rs

```diff
@@ -53,6 +53,7 @@ impl<TK: Tab + Hash + Eq + Send + 'static> TabBar<TK> {
             .with(&ButtonForeground, Color::LIGHTGRAY)
             .with(&ButtonHoverForeground, Color::WHITE)
             .with(&ButtonActiveBackground, Color::GRAY)
+            .with(&WidgetBackground, Color::CLEAR_BLACK)
```

Comment:
Because these buttons are children of the tab bar, and the tab bar has `WidgetBackground` set, the label of the button is inheriting that style and also rendering a background. This isn't ideal for the Select widget, so I will be adjusting this.

To work around the current behavior, we can set the background color to clear.